packages/react-core/src/components/Checkbox/examples/CheckboxRequired.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Checkbox } from '@patternfly/react-core';
+
+export const CheckboxRequired: React.FunctionComponent = () => (
+  <Checkbox
+    label="Required checkbox"
+    isRequired
+    id="required-check"
+    name="required-check"
+    aria-label="Required input"
```

Comment:
Since the checkbox will be linked to the visible text label, we can remove this `aria-label`.